external-applications/quickstart-using-s2i/pom.xml

```diff
@@ -16,6 +16,11 @@
             <artifactId>common</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-junit5</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Why did we need to add this dependency?

Reply:
Because this module require this other dependency `org.junit.platform:junit-platform-launcher:jar:1.7.2:test` that comes with `io.quarkus:quarkus-junit5`

Reply:
How can I reproduce the issue you mention? I removed the `quarkus-junit5` dependency and the test execution worked fine :/